packages/core/src/command/patient-import/patient-import-cloud.ts

```diff
@@ -0,0 +1,182 @@
+import { startDocumentQuery } from "./commands/start-document-query";
+import { createPatient } from "./commands/create-patient";
+import { startPatientQuery } from "./commands/start-patient-query";
+import {
+  PatientImportHandler,
+  StartPatientImportRequest,
+  ProcessPatientImportRequest,
+  ProcessPatientCreateRequest,
+  ProcessPatientQueryRequest,
+} from "./patient-import";
+import { createPatientPayload } from "./patient-import-shared";
+import { Config } from "../../util/config";
+
+const region = Config.getAWSRegion();
+
+const lambdaClient = makeLambdaClient(region);
+const sqsClient = new SQSClient({ region });
+
+export type ProcessPatientImportEvemtPayload = Omit<
+  ProcessPatientImportRequest,
+  "jobStartedAt" | "s3BucketName" | "processPatientCreateQueue"
+>;
+export type ProcessPatientCreateEvemtPayload = Omit<
+  ProcessPatientCreateRequest,
+  "s3BucketName" | "processPatientQueryQueue" | "waitTimeInMillis"
+>;
+export type ProcessPatientQueryEvemtPayload = Omit<
+  ProcessPatientQueryRequest,
+  "s3BucketName" | "waitTimeInMillis"
+>;
+
+export class PatientImportHandlerCloud implements PatientImportHandler {
+  async startPatientImport({
+    cxId,
+    facilityId,
+    jobId,
+    processPatientImportLambda,
+    rerunPdOnNewDemographics = true,
+    dryrun = false,
+  }: StartPatientImportRequest): Promise<void> {
+    const processPatientImportRequest: ProcessPatientImportEvemtPayload = {
+      cxId,
+      facilityId,
+      jobId,
+      rerunPdOnNewDemographics,
+      dryrun,
+    };
+    await lambdaClient
+      .invoke({
+        FunctionName: processPatientImportLambda,
+        InvocationType: "Event",
+        Payload: JSON.stringify(processPatientImportRequest),
+      })
+      .promise();
+  }
+
+  async processPatientImport({
+    cxId,
+    facilityId,
+    jobId,
+    jobStartedAt,
+    s3BucketName,
+    processPatientCreateQueue,
+    rerunPdOnNewDemographics,
+    dryrun,
+  }: ProcessPatientImportRequest): Promise<void> {
+    await createJobRecord({
+      cxId,
+      jobId,
+      data: { jobStartedAt },
+      s3BucketName,
+    });
+    const patients = await validateAndParsePatientImportCsvFromS3({
+      cxId,
+      jobId,
+      s3BucketName,
+    });
+    if (dryrun) return;
+    await Promise.allSettled(
+      patients.map(async patient => {
+        const patientPayload = createPatientPayload(patient);
+        const processPatientCreateRequest: ProcessPatientCreateEvemtPayload = {
+          cxId,
+          facilityId,
+          jobId,
+          patientPayload,
+          rerunPdOnNewDemographics,
+        };
+        await sqsClient.sendMessageToQueue(
+          processPatientCreateQueue,
+          JSON.stringify(processPatientCreateRequest),
+          {
+            fifo: true,
+            messageDeduplicationId: uuidv7(),
+            messageGroupId: cxId,
+          }
+        );
+      })
+    );
```

Comment:
We prob need some error handling code here, so we know for example which patients need reprocessing in case sending message to SQS fails.